shoop/notify/static_src/script-editor.js

```diff
@@ -78,7 +78,7 @@ function Controller() {
         }
     };
     ctrl.setStepItemEditorState = function(state) {
-        if(!!state) {
+        if(state) {
```

Comment:
Oh, that's more of a code style thing (which JSCS handles). Eslint deals with semantics, not form. (And with JavaScript, missing or extraneous semicola are a semantic thing, thanks to ASI.)